README.md

```diff
@@ -12,14 +12,13 @@ Installing Node and npm
 
 ## Introduction
 
-So you pumped up about Node? Me too. But before we can go any further, we need to install it on our system. 
+So you pumped up about Node? Me too. But before we can go any further, we need to install it on our system.
 
 ## Instructions
 
 1. Install Node v5.1.0 with one of the methods listed below
 2. Install npm v2.14.15 (typically comes with Node so you rarely if ever need to install it separately)
-3. Check versions 5.1.0 and 2.14.15 for Node and npm respectively 
-
+3. Check versions 5.1.0 and 2.14.15 for Node and npm respectively
 
 ### One-Click Installers
 
```

Comment:
I would add context for why you want to install with a tar file, from a github repo, why want multiple versions of node.